app/views/groups/samples/index.html.erb

```diff
@@ -7,7 +7,7 @@
   <%= component.with_buttons do %>
     <div class="flex items-center space-x-2">
       <% if allowed_to?(:submit_workflow?, @group) %>
-        <%= link_to pipeline_selection_workflow_executions_submissions_path, data: { turbo_stream: true, controller: "action-link", action_link_required_value: 1 },
+        <%= link_to pipeline_selection_workflow_executions_submissions_path(:namespace_id => @group.id), data: { turbo_stream: true, controller: "action-link", action_link_required_value: 1 },
```

Comment:
Can you change the `:namespace_id => @group.id` to `namespace_id: @group.id`?

Reply:
Removed in 2810a0f4913b41287e4c7fa1bc2aed4cd2e5a4ee